### PR TITLE
post: add js-libp2p 0.25 release post

### PR DESCRIPTION
**Pull request by jacobheun**
Libp2p 0.25 is out! Let's 🌮 about it.

### CODE REVIEW

File: content/post/81-js-libp2p-0.25.md [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+# 🔦 Highlights
+
+Libp2p 0.25 is already in the new [js-ipfs 0.35 release](https://github.com/ipfs/js-ipfs/issues/1826). If you're using js-ipfs, you'll get these changes immediately if you upgrade there!
+
+## ☎️ Auto Dial
+Libp2p will now auto dial to discovered peers by default. This can be disabled in configuration if you need to do any specialized dialing. Discovered peers will only be dialed if we are under our min peers threshold, which is configured for the Connection Manager. You can read more about [Peer Discovery and Auto Dial here](https://github.com/libp2p/js-libp2p/blob/v0.25.0/PEER_DISCOVERY.md).
+
+## ✂️ Better Connection Management
+The Libp2p Switch was doing a bad job of letting the Connection Manager know just how many connections there were. This resulted in the Connection Manager thinking there were far fewer connections than there actually were. This has been fixed, and now the Connection Manager is back to keeping your router safe from the connection plague.
```
Comment (alanshaw) [inline content/post/81-js-libp2p-0.25.md]: I'd rephrase this around the positive, i.e. remove "The Libp2p Switch was doing a bad job" and focus on the fact that there were bugs that were fixed.

File: content/post/81-js-libp2p-0.25.md [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+
+## 📔 Blacklisting Undialable Nodes
+Libp2p will now blacklist nodes that aren't dialable. The blacklisting includes an exponential backoff with a random jitter. This along with auto dial, help to ensure libp2p is spending its connection resources wisely. You can see the configuration options in the [libp2p-switch readme](https://github.com/libp2p/js-libp2p-switch/tree/v0.42.9#create-a-libp2p-switch).
```
Comment (alanshaw) [inline content/post/81-js-libp2p-0.25.md]:
```suggestion
Libp2p will now blacklist nodes that aren't dialable. The blacklisting includes an exponential backoff with a random jitter. This along with auto dial, helps to ensure libp2p is spending its connection resources wisely. You can read more about the configuration options in the [libp2p-switch readme](https://github.com/libp2p/js-libp2p-switch/tree/v0.42.9#create-a-libp2p-switch). 
```

File: content/post/81-js-libp2p-0.25.md [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+
+## 📔 Blacklisting Undialable Nodes
+Libp2p will now blacklist nodes that aren't dialable. The blacklisting includes an exponential backoff with a random jitter. This along with auto dial, help to ensure libp2p is spending its connection resources wisely. You can see the configuration options in the [libp2p-switch readme](https://github.com/libp2p/js-libp2p-switch/tree/v0.42.9#create-a-libp2p-switch). 
+
+## 🔢 Dial Queues
+Libp2p Switch now has better queuing for dials! Previously, the only limiting of dials were done per peer. Now, all dials will enter a global queue so that we can better manage connection attempts. The current default limit is 100 parallel dials, but you can configure this to your needs. If a peer is already connected, any calls to `.dial()` or `.dialProtocol()` skip the line and will be executed immediately. Any dials done as a result of Auto Dial will take a back seat.
```
Comment (alanshaw) [inline content/post/81-js-libp2p-0.25.md]:
```suggestion
Libp2p Switch now has better queuing for dials! Previously, the only limiting of dials was done per peer. Now, all dials will enter a global queue so that we can better manage connection attempts. The current default limit is 100 parallel dials, but you can configure this to your needs. If a peer is already connected, any calls to `.dial()` or `.dialProtocol()` skip the line and will be executed immediately. Any dials done as a result of Auto Dial will take a back seat.
```

File: content/post/81-js-libp2p-0.25.md [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+## ⚖️ Smaller than ever
+Since the 0.24 release, libp2p has dropped some weight. The bundle size is now ~42% smaller!
```
Comment (alanshaw) [inline content/post/81-js-libp2p-0.25.md]:
```suggestion
Since the 0.24 release, libp2p has dropped some weight. The bundle size is now **~42%** smaller!
```

File: content/post/81-js-libp2p-0.25.md (hunk 1 above)
Comment (jacobheun) [inline content/post/81-js-libp2p-0.25.md]: I agree, how about:

```suggestion
We fixed an issue with Libp2p Switch that resulted in the Connection Manager thinking there were far fewer connections than there actually were. The Connection Manager is back to keeping your router safe and your connection count within its [configured range](https://github.com/libp2p/js-libp2p-connection-manager/tree/v0.1.0#create-a-connectionmanager).
```
